mmv1/templates/terraform/examples/cloudbuild_trigger_repo.tf.erb

```diff
@@ -0,0 +1,72 @@
+terraform {
+    required_providers {
+        google-beta = {}
+        google = {}
+    }
+}
+
+provider "google-beta" {}
+
+resource "google_secret_manager_secret" "github-token-secret" {
+    project = "{{project}}"
+    secret_id = "github-token-secret"
+
+    replication {
+        automatic = true
+    }
+}
+
+resource "google_secret_manager_secret_version" "github-token-secret-version" {
+    secret = google_secret_manager_secret.github-token-secret.id
+    secret_data = file("my-github-token.txt")
+}
+
+data "google_iam_policy" "p4sa-secretAccessor" {
+    binding {
+        role = "roles/secretmanager.secretAccessor"
+        // Here, {{projectNum}} is the Google Cloud project number for {{project}}.
+        members = ["serviceAccount:service-{{projectNum}}@gcp-sa-cloudbuild.iam.gserviceaccount.com"]
+    }
+}
+
+resource "google_secret_manager_secret_iam_policy" "policy" {
+  project = google_secret_manager_secret.github-token-secret.project
```

Comment:
We supply a project automatically, and you can omit the field unless the project is defined within the same configuration (same for all occurrences)